lib/renderer/web-view/web-view-attributes.js

```diff
@@ -141,18 +141,15 @@ class GuestInstanceAttribute extends WebViewAttribute {
     if (this.webViewImpl.webviewNode.hasAttribute(this.name)) {
       return parseInt(this.webViewImpl.webviewNode.getAttribute(this.name))
     }
-    return undefined
   }
 
   // Sets the attribute's value.
   setValue (value) {
     if (!value) {
-      return this.webViewImpl.webviewNode.removeAttribute(this.name)
-    }
-    if (isNaN(value)) {
-      return
+      this.webViewImpl.webviewNode.removeAttribute(this.name)
+    } else if (!isNaN(value)) {
```

Comment:
I was hope to not change any behavior in this pull request, would be up for doing this in a follow on where we get a failing spec first and then update it 👍